scripts/benchmark_all.sh

```diff
@@ -74,18 +74,18 @@ do
   for benchmark in $benchmarks
   do
     echo "Running $benchmark for $commit... (single-threaded)"
-    ( ${build_folder}/$benchmark -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.log"
+    ( ${build_folder}/$benchmark --dont_cache_binary_tables -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.log"
```

Comment:
Add case to python-based TPC-H Test?

Reply:
Checking of the clustering config? Or what exactly do you mean?

Reply:
Just change one of the TPC-H cases to use clustering and make sure that "Sorting" is printed